java/arcs/core/data/Plan.kt

```diff
@@ -45,7 +46,8 @@ open class Plan(
         val storageKey: StorageKey,
         val mode: HandleMode,
         val type: Type,
-        val ttl: Ttl? = null
+        val ttl: Ttl? = null,
+        val annotations: List<Annotation>? = emptyList()
```

Comment:
Keep in mind these are not being serialized, nor sent to ArcHosts, because neither the Allocator serialization code, nor Parcelables were updated.